src/CSConsole/ScriptInteraction.cs

```diff
@@ -31,9 +31,18 @@ public static void Stop(Coroutine coro)
 
         public static void Copy(object obj)
             => ClipboardPanel.Copy(obj);
+        
+        public static void Copy(object obj, int index)
```

Comment:
I think this function is not used and should be removed.

Reply:
This method allows you to replace an object at any index with a different object. It probably isn't necessary because I introduced the ```Clipboard``` object in the Console, I just thought maybe it would be useful to have.
If this needs to change, just lemme know and we can figure something out.